test/complexNames.wasm.fromBinary

```diff
@@ -0,0 +1,11 @@
+(module
+ (type $0 (func))
+ (export "$foo\20\28.bar\29" (func $1))
```

Comment:
Hmm, are we escaping both the export and the function name identically? If so then we would be regressing compared to before, as reading and writing a binary won't preserve export names?

Reply:
It just create this name from wast I typed in the complexNames.wast below -- I can retype it unescaped.

Reply:
I thought this file was the result of `wasm-dis tests/complexNames.wasm`, that is, printing out the binary you added to the test suite?

If you added both a wast and a wasm, I might be better to give them different names and not just differentiate them by the suffix - can be confusing.

Reply:
fixed